sycl/source/platform.cpp

```diff
@@ -56,19 +56,39 @@ std::vector<platform> platform::get_platforms() {
 
 backend platform::get_backend() const noexcept { return impl->getBackend(); }
 
+#ifdef __INTEL_PREVIEW_BREAKING_CHANGES
 template <typename Param>
-typename detail::is_platform_info_desc<Param>::return_type
+detail::ABINeutralT_t<
+    typename detail::is_platform_info_desc<Param>::return_type>
+platform::get_info_impl() const {
+  return detail::convert_to_abi_neutral(impl->template get_info<Param>());
+}
```

Comment:
Likewise, this should be enough for both modes.

Reply:
Thanks. Done.